okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -151,7 +151,7 @@ private static ResponseBody buffer(MediaType mediaType, byte[] bodyBytes) {
 
     @Override
     public void enqueue(Callback callback) {
-        DeferredTracer tracer = new DeferredTracer();
+        DeferredTracer tracer = new DeferredTracer("acquire-limiter");
```

Comment:
Would be nice if we prefixed this as well

Reply:
What do you want to prefix this with? `OkHttp: `?

Reply:
Yep, we should group by component, not necessarily class